doc/_static/mpl.css

```diff
@@ -19,7 +19,7 @@ body {
 }
 
 a {
-    color: #CA7900;
+    color: #CA7900 !important;
```

Comment:
the reason for this was to make the inline links show up within sphinx-gallery pages (they weren't showing up before). I didn't realize this was going to mess up the other links as well. A better way for me to have done this would be to add a new rule instead:

```
div.highlight-python a {
    color: #CA7900 !important;
}
```
want me to add this commit to the toolkits PR?